tasks/libs/junit_upload.py

```diff
@@ -16,6 +16,7 @@
 REPO_NAME_PREFIX = "github.com/DataDog/datadog-agent/"
 DATADOG_CI_COMMAND = ["datadog-ci", "junit", "upload"]
 JOB_URL_FILE_NAME = "job_url.txt"
```

Comment:
❓ question
Looks like you no longer use job_url.txt file, do we still need this variable?

Reply:
`job_url.txt` is not present in the kitchen tests indeed, but still produced by the regular unit tests (which use the [produce_junit_tar](https://github.com/DataDog/datadog-agent/blob/b374296d1de21b9dc5c9fda74bbbfa6c0f90ef4b/tasks/libs/junit_upload.py#L153-L184) function).